### DISCUSSION TITLE
New Destination: IBM DB2

### DISCUSSION THREAD

**Discussion by georgmoss** [airbytehq/airbyte#36094] [New Connector Request]
## Tell us about the new connector you’d like to have
IBM DB2 is available as source. That is good but we primarily need the connector as a destination. Perhaps it could be a quite small development since it already exists as a source connector?

## Describe the context around this new connector
Me and my team are running our Data Warehouse in IBM DB2 Warehouse on Cloud. A great database for analytical needs, comparable with Snowflake. We are constantly developing new data pipelines to integrate new systems and collecting data. Some of our integrations are run with short intervals like every 5 minute, some are run every hour and other jobs once a day.

We would need to connect to the database like with JDBC or similar. Like we can do with SQL tools like DBeaver and Squirrel. They use the db2jcc.jar or db2jcc4.jar drivers.

## Describe the alternative you are considering or using
We really hope that we can get our cloud data warehouse functioning as a destination. If not we will probably investigate if we can create the connector ourself and at the same time research other alternative tools for our integrations.

BR,
David at ESS

**Comment by georgmoss** [comment]
Hello. Any commments on this request?

**Comment by misteryeo** [comment]
**Issue was linked to Harvestr Discovery:**  [New Destination: IBM DB2](https://app.harvestr.io/components/0/list/45XEeWWrL)

**Comment by orvischoinierec** [comment]
Do you happen to have any updates on this feature? We need a DB2 destination to send data to an IBM Sailfish appliance and would love to take advantage of Airbyte. I see a Discovery item logged in another system, but I do not have any visibility.

[a comment by an automated account is not shown]
